File: S03-conceptos-basicos/CL12-Mensajes/app/src/main/java/com/udemyandroid/mensajes/MainActivity.java
package com.udemyandroid.mensajes;

import android.support.design.widget.Snackbar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {
    Button btnToast, btnSnackbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnToast = findViewById(R.id.buttonToast);
        btnSnackbar = findViewById(R.id.buttonSnackBar);

        // Evento click
        btnToast.setOnClickListener(this);
        btnSnackbar.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        if (v.getId() == R.id.buttonToast) {
            Toast.makeText(this, "Mostrando Toast", Toast.LENGTH_LONG).show();
        } else if(v .getId() == R.id.buttonSnackBar) {
            Snackbar.make(v, "Mostrando Snackbar", Snackbar.LENGTH_LONG).show();
        }
    }
}

File: S13-kotlin/CL06-EventoClick/app/src/main/java/com/androidavanzado/cl06_eventoclick/MainActivity.kt
package com.androidavanzado.cl06_eventoclick

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var contador = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonContador.setOnClickListener { view ->
                contador++
                Toast.makeText(this, "Click en contador: $contador", Toast.LENGTH_LONG).show()
        }
    }
}
